scout/adapter/mongo/event.py

```diff
@@ -223,25 +232,35 @@ def add_phenotype(
 
             phenotype_id = hpo_obj["_id"]
             description = hpo_obj["description"]
-            if phenotype_id not in existing_terms:
-                phenotype_term = dict(phenotype_id=phenotype_id, feature=description)
-                phenotype_terms.append(phenotype_term)
+            if phenotype_id in existing_terms:
+                continue
 
-                LOG.info(
-                    "Creating event for adding phenotype term for case"
-                    " {0}".format(case["display_name"])
-                )
+            phenotype_term = dict(phenotype_id=phenotype_id, feature=description)
+            if phenotype_inds:
+                phenotype_term["individuals"] = [
+                    {
+                        "individual_id": ind.split("|")[0],
+                        "individual_name": ind.split("|")[1],
+                    }
+                    for ind in phenotype_inds
+                ]
+            phenotype_terms.append(phenotype_term)
 
-                self.create_event(
-                    institute=institute,
-                    case=case,
-                    user=user,
-                    link=link,
-                    category="case",
-                    verb="add_phenotype",
-                    subject=case["display_name"],
-                    content=phenotype_id,
-                )
+            LOG.info(
+                "Creating event for adding phenotype term for case"
+                " {0}".format(case["display_name"])
+            )
+
+            self.create_event(
```

Comment:
Please considering make the event texts more informative now when there is more granularity in how HPO terms can be applied. The current event text is `<user> added HPO term for 643594 at 2021-07-22`, it could instead be something like

`<user> added HPO term <term> to <individuals> at 2021-07-22`

Reply:
That's a really good idea. I'll see what can be done! Thanks for the review 🙏🏻